Fix language and typos (#979)
grafana/grafana-infinity-datasource: docs/sources/references/transformations.md

# Transformations

Server-side transformations are introduced in Infinity 2.0. You can perform basic data manipulation in the server side.

To perform server-side transformations, you need to add a query type of **transformations**. Once this added, **this will perform server side transformation over all the previous infinity queries with backend parser**.

> > **Warning**: Infinity Transformations/Server-side transformations are available only for Infinity data sources or Infinity queries with backend parsers.

Reviewer: I think this is actually a note not a warning but you can replace with "warning" if you disagree

```suggestion
{{< admonition type="note" >}}
Infinity Transformations/Server-side transformations are available only for Infinity data sources or Infinity queries with backend parsers.
{{< /admonition >}}
```

### Computed Column

Appends a new column based on expression.

### Summarize / Group by

Group by results based on aggregation and dimension.

> > **Note**: All these transformations are done post-processing. Once server responds with data, Infinity backend client manipulates the data. If your API supports server-side transformations, use those instead of this.

Reviewer: ```suggestion
{{< admonition type="note" >}}
All these transformations are done after processing.
After the server responds with data, the Infinity backend client manipulates the data.
If your API supports server-side transformations, use those instead.
{{< /admonition >}}
```
